### DISCUSSION TITLE
personalise the font properties

### DISCUSSION THREAD

**Discussion by KALMOGO** [vmware/clarity#6543] [Q&A]
I am not able to change the font-size when I am using the cds-password or cds-input tag 
because the text size is too small. Please could you help me

**Comment by gnomeontherun** [comment]
On the https://clarity.design/core-components/input/api/ you can see the properties to customize as necessary. You would apply the CSS properties onto the element as a style definition (or could be done in your CSS).

```html
<cds-input style="--font-size: 1.2rem">...</cds-input>
```

```css
cds-input {
  --font-size: 1.2rem;
}
```
